src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala

```diff
@@ -34,6 +34,7 @@ class FileService(
     avMetadataService: AntivirusMetadataService,
```

Comment:
I don't think the `consignmentRepository` is being used anymore in the service so can be removed from the constructor.

Reply:
Yep it's not being used. I've removed it.